docker/start.sh

```diff
@@ -6,13 +6,12 @@ envsubst < .env.template  > .env.example
 
 cp .env.example .env
 
-sh /etc/wait-for $DB_HOST:$DB_PORT
-
 php artisan key:generate --force
 php artisan storage:link || true
 
 if [ ${AUTO_INSTALL} == "true" ]
 then
+    wait-for-it $DB_HOST:$DB_PORT
```

Comment:
Can we add a timeout option here. I think the default timeout is 15 seconds. But for mysql container to initialise it would take more than 15 seconds. 
Example: `wait-for-it --timeout=300 $DB_HOST:$DB_PORT`

Reply:
We can do that.

You are right about the 15 seconds. From what I observed, it keeps extending if the container is not up.

So, if we change it to 300, I think the extension would be also 300 I guess (not sure tho). Should we go for it or change it to some less interval?

WDYT @mohan-13?

Reply:
Choosing any interval is fine. Only concern is it should wait until db container boots up.

Reply:
It makes sense to me, I added the timeout flag. It would be great if you can squash and merge the PR in the end.
Thanks @mohan-13